.github/workflows/docs.yml

```
@@ -11,6 +11,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Install tooling
+        run: |
+          apt-get install -y protobuf
```

```suggestion
          apt-get install -y protobuf-compiler
```

I'm not sure but the [official doc](https://grpc.io/docs/protoc-installation/) says that the package has a slightly different name

---

Totally, this is also what I did in https://github.com/paritytech/srtool/pull/50/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R35 🤦 and confirmed fixing the issues.